tests/test_HomSap.py

```diff
@@ -31,13 +31,13 @@ def test_recombination_rates(self):
         # compare the results to the current recombination rates for each chromosome
         genetic_map = "HapMapII_GRCh37"
         species = stdpopsim.get_species("HomSap")
-        for chrom in self.genome.chromosomes:
+        for chrom in self.genome.chromosomes[2:]:
             if chrom.id == "chrY":
```

Comment:
Hmm... I don't think we have a `chrY` anymore, only a `Y`. So this probably doesn't get executed anymore.

Reply:
Changed to "Y", and confirmed it's running now.